CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.6.0] - 2024-11-13
+### Added
+- Adds support for Ruby 3.2
```

Comment:
I believe `File.exists?` was dropped in Ruby 3.0

Reply:
@schlick https://rubyreferences.github.io/rubychanges/3.2.html#removals

Reply:
Indeed you are right! I was basing it on this resource: https://rubyapi.org/3.0/o/file where it doesn't list `exists?` as a method but it is definitely still there in 3.1 and removed in 3.2